CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ the code was deployed.
 
```

Comment:
This looks like a nice article talking about doing this sort of thing: https://testfully.io/blog/api-health-check-monitoring/

The part about specifying no-cache when hitting your endpoint is a good idea to remember when you work on the PA side of thing.

Reply:
Oh that's a really good idea! I'll make sure to imnclude that in the request headers in PA.